Freenas/.github/workflows/pssa.yml

```diff
@@ -0,0 +1,13 @@
+name: CI
+on: [pull_request]
+jobs:
+  lint:
+    name: Run PSSA
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: lint
+      uses: devblackops/github-action-psscriptanalyzer@master
+      with:
+        repoToken: ${{ secrets.GITHUB_TOKEN }}
+        settingsPath: pssa_settings.psd1
```

Comment:
May be create a pssa_settings.psd1 on the root of your github project for Include or exclude some warning... ( https://github.com/PowerShell/PSScriptAnalyzer#explicit )